lightwood/helpers/torch.py

```diff
@@ -27,6 +27,25 @@ class LightwoodAutocast:
     """
     Equivalent to torch.cuda.amp.autocast, but checks device compute capability
     to activate the feature only when the GPU has tensor cores to leverage AMP.
+
+    **Attributes:**
+
+    * `active` (bool): Whether AMP is currently active.
```

Comment:
Mention that this is at the class level, not instance. This is a pretty important implementation detail.